api/src/main/java/jakarta/config/Config.java

```diff
@@ -30,27 +28,73 @@
  */
 public interface Config {
     /**
-     * Return all the currently registered {@link jakarta.config.spi.ConfigSource sources} for this configuration.
+     * Context related to the root configuration instance.
```

Comment:
```suggestion
     * Return the context related to the root configuration instance.
```

Reply:
The "return" word is redundant in documentation of getters. Is this defined in Jakarta guidelines?

Reply:
I am not aware of any guidelines. It just reads well. I don't have strong opinion on this.